gapminders/gapminders.ipynb

```diff
@@ -83,20 +83,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "@pn.cache\n",
     "def get_dataset():\n",
-    "    url = 'https://raw.githubusercontent.com/plotly/datasets/master/gapminderDataFiveYear.csv'\n",
-    "    return pd.read_csv(url)"
+    "    return pd.read_csv('data/gapminders.csv')\n",
```

Comment:
The `get_dataset` function was useful when decorated with `pn.cache` and fetching data remotely. I think it's no longer required, you could just replace it with `dataset = d.read_csv('data/gapminders.csv')`.

Reply:
I see, that makes sense.

Reply:
@jtao1 Not only is this link broken, but we are also not using cache in this cell below this anymore. Checking links was one of the items in the checklist on this PR description. Please review the checklist again in greater detail. When you have confirmed that you have completed all the steps, message me to continue with the review. 

![image](https://github.com/holoviz-topics/examples/assets/6613202/cae8f827-ba00-423e-b1f5-efd1a88632f4)

Reply:
there's also still a pn.cache decorator elsewhere that should be removed

Reply:
I've made a commit to remove that link since it no longer is useful and I've removed the pn.cache decorator.